server/src/main/java/io/crate/expression/predicate/IsNullPredicate.java

```diff
@@ -132,7 +143,7 @@ public static Query refExistsQuery(Reference ref, Context context) {
         if (storageSupport == null && ref instanceof DynamicReference) {
             return Queries.newMatchNoDocsQuery("DynamicReference/type without storageSupport does not exist");
         } else if (ref.hasDocValues()) {
-            return new ConstantScoreQuery(new FieldExistsQuery(field));
+            return new FieldExistsQuery(field);
```

Comment:
Why did we have a `ConstantScoreQuery` here in the first place?

Reply:
Not sure but it should be required, we set `ScoreMode` when we create the weight to avoid scoring if it isn't needed.